From 6c75901e92cc48997f92f8e8202abdbce0453f38 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Fri, 28 Apr 2023 15:37:41 -0700
Subject: [PATCH] Disable tests

These are run at setup, breaking the build unless all the python deps
exist, which they don't.
---
 meson.build | 38 ++++++++++++++++++++------------------
 1 file changed, 20 insertions(+), 18 deletions(-)

diff --git a/meson.build b/meson.build
index 66cda8b5a8a9..61b531bc4879 100644
--- a/meson.build
+++ b/meson.build
@@ -75,6 +75,7 @@ if get_option('unit_testing')
 endif
 
 #FIXME: exclude doc/env/
+if (false)
 test('linting',
      pyflakes,
      timeout: 100,
@@ -97,6 +98,7 @@ test('unit-tests',
      args: ['run', '-a', '-m', 'pytest', '-s', '-v', '--cov-append', meson.current_source_dir()],
      timeout: 600,
      env: test_env)
+endif
 
 #TODO: the coverage section should probably be cleaned up a bit
 if get_option('b_coverage')
